wfl/Database/MongoDBManager.cs

```diff
@@ -26,6 +27,16 @@ public MongoDBManager()
             await collection.InsertOneAsync(r);
         }
 
+        public async Task<RestaurantDto> UpdateRestaurantNameAsync(string id, string name)
```

Comment:
It would make more sense to have one UpdateRestaurantAsync function that takes a RestaurantDto object as a parameter and updates multiple fields based upon what was supplied in the request.